### ISSUE TITLE
case sensitivity

### ISSUE THREAD

**Issue by chetanmotamarri** [wbish/jsondiffpatch.net#29]

**Comment by chetanmotamarri** [comment]
Any update on this one ?

**Comment by wbish** [comment]
No, feel free to add these features and submit a PR. Unfortunately, I don't have time to work on these.

For the first one I think maybe the right approach would be to expose an comparer interface people can implement and pass into JsonDiffPatch -- that way we can support weird cases like (int)1 == (string)"1"

The second one exists in the original JsonDiffPatch library already and something we should definitely add. Search for "detectMove" in benjamine/jsondiffpatch (https://github.com/benjamine/jsondiffpatch/blob/0c4323e9bff23ae231f4bff231ceed4df2b48be5/src/filters/arrays.js#L192)
